charts/fybrik/files/taxonomy/base_taxonomy.json

```diff
@@ -0,0 +1,190 @@
+{
+  "title": "taxonomy.json",
+  "definitions": {
+    "Action": {
+      "type": "object",
+      "required": [
+        "name"
+      ],
+      "properties": {
+        "name": {
+          "$ref": "#/definitions/ActionName"
+        }
+      },
+      "additionalProperties": true
+    },
+    "ActionName": {
+      "type": "string"
+    },
+    "AppInfo": {
+      "type": "object",
+      "additionalProperties": true
+    },
+    "AssetID": {
+      "description": "Asset ID of the asset to be queried in the catalog",
+      "type": "string"
+    },
+    "AttributeType": {
+      "type": "string",
+      "enum": [
+        "numeric",
+        "string",
+        "bool"
+      ]
+    },
+    "Capability": {
+      "type": "string"
+    },
+    "Connection": {
+      "description": "Name of the connection to the data source",
+      "type": "object",
+      "required": [
+        "name"
+      ],
+      "properties": {
+        "name": {
+          "$ref": "#/definitions/ConnectionType",
+          "description": "Name of the connection to the data source"
+        }
+      },
+      "additionalProperties": true
+    },
+    "ConnectionType": {
+      "description": "Name of the connection type to the data source",
+      "type": "string"
+    },
+    "DataFlow": {
+      "type": "string",
+      "enum": [
+        "read",
+        "write",
+        "delete",
+        "copy"
+      ]
+    },
+    "DataFormat": {
+      "description": "Format of the data",
```

Comment:
Isn't it more accurate to say that the DataFormat is the format the workload is reading/writing the data.  It's not necessarily the format in which it is stored.  Correct?

Reply:
Correct, changed the description